.github/workflows/tests.yml

```diff
@@ -148,10 +148,10 @@ jobs:
         # Set of versions to use for the matrix tests.
         os: ["24.04"]
         go: ["1.22.x"]
-        microceph: ["latest/edge"]
-        microovn: ["latest/edge"]
-        lxd: ["5.21/edge"]
```

Comment:
While we should have tests using the stable track, I think we should also keep testing against `/edge` to catch regressions early.

Reply:
I think it would be great if we run the same set of tests using the `x/edge` and `x/candidate` channels. 
Maybe let's do this as part of moving to the Canonical runners? It will be a lot of long running jobs.

Reply:
Or run them on a daily/weekly schedule.

Reply:
https://github.com/canonical/microcloud/issues/493